tests/acceptance/include_config_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import pytest
+from webtest import TestApp
+
+from .app import main
+
+
+@pytest.fixture
+def settings():
+    dir_path = 'tests/sample_schemas/include_config_app/'
+    return {
+        'pyramid_swagger.schema_directory': dir_path,
+        'pyramid_swagger.enable_request_validation': True,
+        'pyramid_swagger.enable_swagger_spec_validation': True,
+        'pyramid_swagger.swagger_versions': ['1.2', '2.0'],
+    }
+
+
+@pytest.fixture
+def test_app_with_no_include_conf(settings):
```

Comment:
"include conf" or "include_config" is the old name. Think this module needs an update